Source/santactl/Commands/SNTCommandStatus.m

```diff
@@ -91,23 +91,14 @@ - (void)runWithArguments:(NSArray *)arguments {
   }];
 
   // Database counts
-  __block int64_t eventCount = -1;
-  __block int64_t binaryRuleCount = -1;
-  __block int64_t certRuleCount = -1;
-  __block int64_t teamIDRuleCount = -1;
-  __block int64_t signingIDRuleCount = -1;
-  __block int64_t compilerRuleCount = -1;
-  __block int64_t transitiveRuleCount = -1;
-  [rop databaseRuleCounts:^(int64_t binary, int64_t certificate, int64_t compiler,
-                            int64_t transitive, int64_t teamID, int64_t signingID) {
-    binaryRuleCount = binary;
-    certRuleCount = certificate;
-    teamIDRuleCount = teamID;
-    signingIDRuleCount = signingID;
-    compilerRuleCount = compiler;
-    transitiveRuleCount = transitive;
+  __block struct RuleCounts ruleCounts;
+  memset(&ruleCounts, NSUIntegerMax, sizeof(ruleCounts));
```

Comment:
```suggestion
  __block struct RuleCounts ruleCounts = {0};
```

Reply:
Unless I'm missing a reason this is initializing to NSUintegerMax instead of 0.

Reply:
Previously, all counts were initialized to -1. This kinda made sense because the values were ints (even though data was obtained as an `NSUInteger`). However if there was some XPC failure, the default of -1 would get displayed indicating an error. 

I made the types of the struct match the method signatures, but I now think that isn't quite right. The query returns a signed `long`, and that gets converted to an `NSUInteger`. I think I should instead fix the rule count methods to return long and then make this struct match those same types. Then we can go back to the `-1` default as well.